src/app/zones/data/index.ts

```diff
@@ -87,7 +87,7 @@ export const ZoneOverview = {
         payload: {},
       })
     }
-    if (!get(insight, 'version.kumaCp.kumaCpGlobalCompatible', 'true')) {
+    if (!get(insight, 'version.kumaCp.kumaCpGlobalCompatible', true)) {
```

Comment:
This one wasn't necessarily a console warning error, and doesn't fix anything, but its an issue in the same vein in that its incorrect but doesn't necessarily have any negative effect in our current app.

---

This should default to a boolean not a string if `kumaCpGlobalCompatible` isn't set. Our `get` utilities isn't very type safe as it always deals with `any` instead of using generics.

I plan to remove our usage of `get` if possible (we only use it in a few places), but if I can't completely get rid of it, then I'll try and convert it to use generics instead of `any` which would probably flag any similar errors like this.